core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -84,11 +84,20 @@
       <login-required-message></login-required-message>
     </div>
     <div class="oppia-login-required" *ngIf="userDetailsLoading || userIsLoggedIn">
+
+      <div *ngIf="activeExplorationId === null && isReviewTranslationsTab() && !isAccomplishmentsTabActive()" class="oppia-language-selector-container">
+        <review-translation-language-selector class="oppia-language-selector"
+        [activeLanguageCode]="languageCode"
+        (setActiveLanguageCode)="onChangeLanguage($event)">
```

Comment:
I think we usually would indent this to align with the top-level attribute, in this case, `class`.

Reply:
Added!